docs/chainlink-keepers/cron-contracts.md

```diff
@@ -0,0 +1,629 @@
+
+![Remix Contract Interface](/images/keepers/cron-3.png)
+
+## Scheduling a Job
+
+To schedule a new job, use the function `createCronJobFromEncodedSpec`. This function has three inputs: 
+
++ The target address of the smart contract you want to automate
++ The [function signature](#function-signature), or the encoding of the function, you want to call 
++ The [encodedCronSpec](#encodedcronspec) encoding of your specified time interval.
+
+Refer to the [Encodings](#encodings) section below to retrieve the [function signature](#function-signature) and the [encodedCronSpec](#encodedcronspec) for your job. 
+
+```json
+<your target address>, <your function specification>, <your encodedCronSpec>
+```
+Replace the values and paste the snippet above into the `createCronJobFromEncodedSpec` function in Remix and execute the function. Once executed, you can call the `getActiveCronJobIDs` to see if your job has been registered. You can execute the `getCronJob` function to view the details of your job played back. You can add multiple jobs on this contract in this fashion, and also use the other in your Job Scheduler to delete jobs or pause and unpause the Scheduler. However, you must register your Job Scheduler with the Chainlink Keepers network for it to be monitored and executed.
+
+
+## Setting up Chainlink Keepers
+
+Since our deployed contract is already [Keeper-compatible](../compatible-contracts), we can immediately register it with Keepers following these [steps](../register-upkeep). Be sure to provide the address of your Job Scheduler contract in **Upkeep address**. For gas limits, you should specify the upper limit of gas your target function will use.
+
+> ❗️ Gas Limits
+>
+> The `KeeperRegistry` enforces a cap for gas used both on-chain and off-chain. See the [Keepers Network Overview](../overview/) for details. The caps are configurable and might change based on user feedback. Be sure that you understand these limits if your use case requires a large amount of gas.
+
+Once you have registered your contract, you can view your Upkeep. The *Upkeep History* will display **Perform Upkeep** for all registered jobs on your Job Scheduler contract. To ensure Chainlink Keepers monitors your Job Scheduler, please ensure you fund your Upkeep.
+
+# Summary
+
+In this tutorial you used the Chainlink Keepers Job Scheduler to automate the execution of your smart contract functions at specific times. 
+
+# Encodings
+
+Encodings allow you to save repeated gas costs of having your contract calculate them during execution.
```

Comment:
```
Encodings allow you to save repeated gas costs of having your contract calculate them during execution.
```
Let's explain this a little more thoroughly. Why would the contract not just accept Cron in its normal format?